contrib/qpl-cmake/uuid/uuid.h

```diff
@@ -0,0 +1,104 @@
+/*
+ * Public include file for the UUID library
```

Comment:
We already have `Poco/UUID.h` to support various uuid operations. Do we really need another one?

Reply:
because contrib/qpl has dependency on uuid with the specific path "uuid/uuid.h", so I have to create uuid.h under contrib/qpl-cmake/uuid/

./contrib/qpl/sources/core-iaa/include/libaccel_config.h
......
#include <uuid/uuid.h>
......

Meanwhile, I simplified the content of uuid.h since the interface required is just  only the uuid_t definition.